cfme/tests/control/test_actions.py

```diff
@@ -108,9 +109,10 @@ def vm(request, provider_mgmt, provider_crud, provider_key, provider_data, small
     if isinstance(provider_mgmt, mgmt_system.RHEVMSystem):
         # RHEV-M is sometimes overloaded, so a little protection here
         try:
-            provider_mgmt.deploy_template(
-                small_template,
-                vm_name=vm_name,
+            deploy_template(
```

Comment:
I guess I meant I didn't think we should have to specify the provider key, because as we already have provider_mgmt, we should use that.